package.json

```diff
@@ -31,6 +31,7 @@
     "tslib": "2.6.0",
     "typescript": "5.1.6",
     "vite": "4.5.3",
+    "cropperjs": "1.6.2",
```

Comment:
Cropper should be implemented like the other editors, without any additional installations of modules.